applications/dashboard/models/class.importmodel.php

```diff
@@ -1784,4 +1787,64 @@ public function UpdateCounts() {
 
       return TRUE;
 	}
+
+	/**
+	 * Verify imported data
+	 */
+	public function VerifyImport() {
+		// When was the latest discussion posted?
+		$LatestDiscussion = $this->SQL->Get('Discussion', 'DateInserted', 'desc', 1);
```

Comment:
This is suboptimal on large tables. What you really want is:

```php
$MaxDate = $this->SQL->Select('DateInserted', 'max', 'MaxDate')->From('Discussion')->Get()->Value('MaxDate');
```

Check the query and you'll see:

```SQL
describe select max(DateInserted)
from GDN_Discussion;
```